src/extension.ts

```diff
@@ -26,9 +31,51 @@ export async function activate (context: vscode.ExtensionContext) {
 	context.subscriptions.push(...subscriptions);
 }
 
-// TO-DO: add call to Hydrate Python script
-function hydrateCluster () {
+/**
+ * This method spawns a subprocess that calls Hydrate, which creates a Fabrikate 
+ * component.yaml file in the same directory as the Hydrate clone.
+ * 
+ * It runs Hydrate on the kubeconfig file associated with the clusters in the sidebar,
+ * and it currently uses the default values for the rest of the options.
+ */
+function hydrateCluster (target?: any) {
+	let kubeconfig = getKubeConfig();
```

Comment:
two things here;

1. you can use a `const` binding instead of a `let` binding, as `kubeconfig` is unlikely to change after it's been retrieved
2. you should validate that the file path from `getKubeConfig` exists